pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/PepperKitUtil.java

```diff
@@ -3,9 +3,7 @@
 import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
```

Comment:
`TestKitUtil` is not changed to `PepperKitUtil`, while lots of its kit functionality is now in the `BaseKitTestUtil`